scipy/stats/_multivariate.py

```diff
@@ -1294,6 +1294,16 @@ def _lnB(alpha):
     return np.sum(gammaln(alpha)) - gammaln(np.sum(alpha))
 
 
+_dirichlet_depr_message = (
+"""
+`dirichlet` is deprecated due to an interface inconsistency: compared to
+other distributions, methods `pdf` and  `logpdf` expect the transpose of the
+input`x`. Please use `multivariate_beta`, which corrects this inconsistency.
+In SciPy 1.11.0, this deprecation warning will be removed and `dirichlet` will
+become an alias for `multivariate_beta`.
+""")
+
+
 class dirichlet_gen(multi_rv_generic):
     r"""A Dirichlet random variable.
```

Comment:
Should we change this to

```suggestion
    r"""A Dirichlet random variable (deprecated, use `multivariate_beta` instead).
```

?